examples/complete/main.tf

```diff
@@ -85,6 +85,8 @@ module "ec2_complete" {
     }
   ]
 
+  create_eip_public = true
```

Comment:
Let's add a way to "create EIP", "bring your own EIP" and use [aws_eip_association](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/eip_association) to manage the relations.